### PR TITLE
POSYDON v2.0.0-pre1

### PR DESCRIPTION
**Pull request by maxbriel**
This PR is the prerelease for the POSYDON v2 code, as described in [Andrews et al. (submitted)](https://ui.adsabs.harvard.edu/abs/2024arXiv241102376A/abstract).

It has been used to produce the reprocessed V1 dataset such that they're compatible with the POSYDON v2 code.
POSYDON v1 grids for v2 code
https://zenodo.org/records/14205146

POSYDON v1 super-Eddington accretion grid for v2 code
https://zenodo.org/records/14216817

### CODE REVIEW

**Comment by maxbriel** [comment]
I had to merge in the "max_prepare_v2_prerelease" branch, such that that PR has been accepted, we can merge this PR into "main."

**Comment by mkruckow** [comment]
@maxbriel, the [Merge branch 'main' into max_requirements](https://github.com/POSYDON-code/POSYDON/pull/451/commits/9bc93b5058434ef8b4dbfbe076e0162e04457bb9) duplicated some lines. Could you please remove the second occurrences again?

File: posydon/grids/post_processing.py [hunk 1]
```diff
@@ -24,29 +29,66 @@
...
+                  "{}: {}".format(MODEL_NAME, e), "InappropriateValueWarning")
+        
+    
+                    
+def post_process_grid(grid, index=None, star_2_CO=True, MODELS=MODELS,
                       single_star=False, verbose=False):
-    """Compute post processed quantitiy of any grid.
+    """Compute post processed quantity of any grid.
 
+    This function post processes any supported grid and computes:
```
Comment (elizabethteng) [inline posydon/grids/post_processing.py:90]: redundant with lines below

**Review by elizabethteng — COMMENTED** [review]
Looks good! We should ask Tassos and Jeff again about the author email tomorrow. 
EDIT: Hope you don't mind that I went ahead and removed the redundant lines

**Comment by celiotine** [comment]
I ran both NN and IF populations with this branch and the new post-processed v1 data. I didn't find any errors apart from those already recorded in the git issues. If we don't plan to address those issues for the pre-release and are ok with adding them in the main v2 release, then I am ok to accept!

**Comment by astroJeff** [comment]
All looks good to me. Let's discuss today. I would like to merge this soon

**Comment by maxbriel** [comment]
I've implemented the requested changes to this PR:
- include PR #421 
- change posydon email
- remove VH diagram `binary` flag that came from main

**Comment by mkruckow** [comment]
Didn't we said, that we are going to increase the default memory for v1 data because the 5GB look to be (too close to be) insufficient?

**Comment by maxbriel** [comment]
+ updated to 7GB

**Review by elizabethteng — APPROVED** [review]

**Review by sgossage — APPROVED** [review]

**Review by mkruckow — APPROVED** [review]

**Review by mkruckow — CHANGES_REQUESTED** [review]
Please update the new version name, we said, everywhere in the setups for conda, pip, e.g. in [conda/meta.yaml](https://github.com/POSYDON-code/POSYDON/pull/451/files#diff-e4db5852c039b631ee11f2e6f3b000f53e0774918ff1ba8f983f428f72b4fe4c)

Additionally, can you update the release page of the documentation: [docs/_source/release-notes/major-updates.rst](https://github.com/POSYDON-code/POSYDON/pull/451/files#diff-ee56f2729b815d39df6ffa1926ca33c5fb7839363d3a11074758683b179fa3e0) and [docs/_source/release-notes/version-history.rst](https://github.com/POSYDON-code/POSYDON/pull/451/files#diff-3502ddb92423efc632c97f9e62c52c89088f96e4de00ecad8fd99d5a75358d79)?

Finally the installation instructions would benefit from an update to the new data set on zenodo: [docs/_source/getting-started/installation-guide.rst](https://github.com/POSYDON-code/POSYDON/pull/451/files#diff-0fefe184744a03da6698d9c1de0b60ee56a48a9627cd5419ae479ef02d15b32f).

**Review by elizabethteng — DISMISSED** [review]

**Comment by elizabethteng** [comment]
failed the following unit tests: 

posydon/unit_tests/utils/test_data_download.py::TestValues::test_value_data_url FAILED [ 68%]
posydon/unit_tests/utils/test_data_download.py::TestValues::test_value_original_md5 FAILED [ 68%]

**Comment by mkruckow** [comment]
> posydon/unit_tests/utils/test_data_download.py::TestValues::test_value_data_url FAILED [ 68%]
> posydon/unit_tests/utils/test_data_download.py::TestValues::test_value_original_md5 FAILED [ 68%]

Those are the unit tests for the data download, which are expected to fail as long as the new Zenodo url and corresponding check sum aren't propagated to the unit tests.
I have prepared the branch [matthias_update_data_download](https://github.com/POSYDON-code/POSYDON/tree/matthias_update_data_download) (it contains the latest changes in [development](https://github.com/POSYDON-code/POSYDON/tree/development) + the update of the data download including the corresponding unit test) to merge into [max_requirements](https://github.com/POSYDON-code/POSYDON/tree/max_requirements) in case you like to include the unit-tests in the v2.0.0-pre1. Otherwise, the pre release should work without the passed unit test. But surely the update of the unit test should be included in PR #456 .
P.S.: Why are the unit tests executed here, strictly speaking their code don't exist in this branch until the current development gets merged in here, see e.g. [continuous_integration.yml](https://github.com/POSYDON-code/POSYDON/blob/max_requirements/.github/workflows/continuous_integration.yml) of this branch.

**Comment by maxbriel** [comment]
I think GitHub is confused due to the presence of two PRs from the same branch:
Here's the actual Github actions report for this run:
https://github.com/POSYDON-code/POSYDON/actions/runs/12202509354
It passed.
Here's the one that it thinks is associated with this PR: https://github.com/POSYDON-code/POSYDON/actions/runs/12202509369

**Comment by maxbriel** [comment]
I've implemented the suggested changes by @mkruckow for the version number and I've changed the documentation to point to the Zenodo community + Github releases page. This should reduce the number of places we have to maintain with accurate project information.

The PR passes the Github action checks: https://github.com/POSYDON-code/POSYDON/actions/runs/12202509354
The failed checks shown here are not for this PR but for #456.

**Review by elizabethteng — APPROVED** [review]

**Review by astroJeff — APPROVED** [review]

**Pull request merged**